client/verta/requirements-unit-tests.txt

```diff
@@ -3,4 +3,5 @@ hypothesis
 pytest >= 4.3
 pytest-xdist >= 1.32
 six >= 1.11
-responses >= 0.22
\ No newline at end of file
+responses >= 0.22
+pydantic
```

Comment:
I realize [I literally asked for exactly this](https://github.com/VertaAI/modeldb/pull/3910/files#r1248305601), but it occurs to me this should be in [`requirements-dev.txt`](https://github.com/VertaAI/modeldb/blob/60c03fefb30bd8f64dd11211ade0fec3ebb7893c/client/verta/requirements-dev.txt) instead 😬